tests/e2e/e2e_test.go

```diff
@@ -415,3 +442,290 @@ var _ = Describe("odoe2e", func() {
 		})
 	})
 })
+
+var _ = Describe("component update odoe2e", func() {
+	appTestName := "testing"
+
+	bootStrapSupervisorURI := "https://github.com/kadel/bootstrap-supervisored-s2i"
+	initContainerName := "copy-files-to-volume"
+	wildflyUri1 := "https://github.com/marekjelen/katacoda-odo-backend"
+	wildflyUri2 := "https://github.com/mik-dass/katacoda-odo-backend"
+	appRootVolumeName := "-testing-s2idata"
+
+	tmpDir, err := ioutil.TempDir("", "odo")
+	if err != nil {
+		Fail(err.Error())
+	}
+
+	Describe("updating the component", func() {
```

Comment:
You need to create odo project/application before running this. It is likely to run into conflicts with the other tests otherwise.

Reply:
ok updating